packages/bridge-ui/src/proof/service.ts

```diff
@@ -86,6 +77,60 @@ class ProofService implements Prover {
 
     return signalProof;
   }
+
+  async GenerateProof(opts: GenerateProofOpts): Promise<string> {
+    const key = ProofService.getKey(opts);
```

Comment:
i think its more idiomatic to use `this.` and not `ProofService.`

Reply:
Since these methods don't change the `instance` and are rather mostly pure (with side-effects) I think it will be more appropriate to have them as `static` methods than `instance` methods, what do you think @cyberhorsey ?

Reply:
Can also make them utils/helpers with better names